packages/mobile/src/invite/actions.ts

```diff
@@ -16,8 +16,17 @@ export enum Actions {
   SKIP_INVITE_FAILURE = 'INVITE/SKIP_INVITE_FAILURE',
 }
 
-export interface Invitees {
-  [tempAddress: string]: string // tempAddress -> e164Number
+export interface InviteDetails {
```

Comment:
@jmrossy Another option to eliminate redundancy between EscrowedPayment and InviteDetails is to eliminate tx related data (timestamp, escrowAmmount, escrowTxId, etc) from InviteDetails and just do a join on the two structures when we need to. What do you think?

Reply:
Yep that works! Up to you, it's not a significant amount of data so should be fine either way

Reply:
Great, I am going that route. All things equal, I'd rather the data sets be MECE.